package.json

```diff
@@ -8,7 +8,7 @@
     "test": "",
     "build": "npm run clean; webpack --env=prod --colors",
     "clean": "rm -rf dist",
-    "dist:server": "pushd dist; python -m SimpleHTTPServer; popd"
+    "dist:server": ". ./scripts/dist_server.sh"
```

Comment:
[ defer, no idea, I thought I tested this and saw that it was needed but now can't repro... ]